index.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@200;500&display=swap"
+      rel="stylesheet"
+    />
+
+    <title>Abdulrahman sharshar</title>
+  </head>
+  <body>
+    <div class="main">
+      <header>
+        <nav>
+          <h1 class="Logo-heading">Welcome</h1>
```

Comment:
- [ ] [REQUIRED] I think you should remove this `h1` from the nav tag and also make it an `h2` or `h3` tag because there is only supposed to be one h1 tag on each page😊. So it will be
```
<h2 class="Logo-heading">Welcome</h2>
<nav>
    <img class="icon" src="images/IconMenu.png" alt="icon" />
</nav>
```